### DISCUSSION TITLE
Custom and Responsive classes

### DISCUSSION THREAD

**Discussion by jrozanski** [primefaces/community#96] [PrimeFlex]
Hi folks, 

Currently, we can add new classes, for example

```main.scss
@import 'primeflex/primeflex.scss';

.l-content-layout {
    @include styleclass('m-0');
    width: 1400px;
}
```

But this class will not be responsive, so can't be used like `md:l-content-layout`. Is there a mechanism to extend PrimeFlex to include custom classes which support various modifiers? This feels natural coming from Tailwind, but I am looking for PrimeFlex way.

I am getting the feeling that's what PrimeFlex 4.0 is about, but maybe there is a trick in 3.x to do it.

**Comment by atkntepe** [comment]
Hi, yes, the 4.0 is all about customization. It will be released soon. We are working on the documentation side at the moment. Some urgent cases caused delays, but we are on it.

**Reply by jrozanski** [reply]
Hi @atakantepe any news of Flex 4 release?
